kernel/syscall.h

```diff
@@ -25,3 +25,4 @@
 #define SYS_isatty 24
 #define SYS_ftruncate 25
 #define SYS_readdir 26
+#define SYS_execve  27
```

Comment:
Could you remove `SYS_exec` and implement `exec` function in user library?

Reply:
As you said, it is needed to implement PATH parsing to get rid of exec syscall and to implement it in userland library.
Apparently, we've start writing environment variable handing facility in  shell program, 
would you please continue this work and implement exec function in ulib by yourself?
To be honest, I am not able to have an interesting about implement the exec function in libc.

Thanks,
Regards,